tests/IntegrationTests/ItemPropertiesSetupTest.php

```diff
@@ -0,0 +1,104 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\Matej\IntegrationTests;
+
+use Lmc\Matej\Model\Command;
+
+/**
+ * @covers ItemPropertiesSetupRequestBuilder
+ * @covers EventsRequestBuilder
+ */
+class ItemPropertiesSetupTest extends IntegrationTestCase
+{
+    /** @test This throws an error in Matej's MGW, but API will return OK */
```

Comment:
In case of description, @depends etc. I'd use multiline phpDoc :). Also in the other cases in this PR.